### DISCUSSION TITLE
Gold ore question

### DISCUSSION THREAD

**Discussion by avescarelo** [Roadhog360/Et-Futurum-Requiem#393] [Improvements]
Hello, I use EtFuturum and Netherliqiuos mods at the same time, both of these mobs add Neher gold ore, but Netherliqious changes Nether vanilla stone to his own with a new texture, for which he has gold ore made. 

And the gold ore from EtFuturum has an old texture, which is why it stands out strangely in the Cave.
I can show you what it looks like in the cave if necessary :d

Can you make a switchable gold ore texture to a new one if this mod is installed? 

Also, it does not apply to this topic, but both mods complement Nether well and duplicate items, plates, mobs, etc., it looks funny. ^-^

![image](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/26678843/4998c402-b545-480e-8630-750021efceab)
![image](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/26678843/02468f01-7153-4803-a4a6-7a47f817879d)

**Comment by avescarelo** [comment]
Two variants 
![image](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/26678843/e2ec38bf-559e-425f-bb41-7c3cb0ba93cb)

**Comment by Roadhog360** [comment]
Use a texture pack, that's what they're for. But both gold ores are not supposed to be used together anyways, turn off the EFR one
